main.js

```diff
@@ -42,7 +55,8 @@ let domString = " "
 <div class="card-body">
   <h5 class="card-title">Please select the sort button to begin</h5>
   <p class="card-text">Today is only the beginning! </p>
- <a  id= "sort" onclick="sort()" href="#" class="btn btn-primary">Start</a>
+ <a  id= "start" onclick="sort()" href="#" class="btn btn-primary">Start</a>
```

Comment:
We don't want to use `onClick` in our raw HTML. We always want to use JS event listeners for this.